samples/basic/blinky/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _blinky-sample:
+.. zephyr:code-sample:: blinky-sample
```

Comment:
Since we are adding this directive, shouldn't it also populate some common fields in the `.rst` where this is placed to ensure all of our samples are consistent in their doc?

Reply:
I am not sure I understand, can you give an example?

Reply:
Yes, why not have this directive populate the title of the sample as well as any additional sample title formatting?

Reply:
But that's what it does already, does it not? Notice how the title and explicit target are both gone in the diff, since the directive is taking care of both now.

Reply:
I guess we could add `:software_requirements:` and `:hardware_requirements` as two other "standard" (and optional) sections for which people could provide details directly in the directive, but I think the added value is pretty limited for anything that's just free text.

Reply:
> But that's what it does already, does it not? Notice how the title and explicit target are both gone in the diff, since the directive is taking care of both now.

Oh right, yes, I realize now I misread this. This is good enough for me then now.